fastly/resource_fastly_service_v1.go

```diff
@@ -1319,7 +1326,7 @@ func resourceServiceV1Create(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*FastlyClient).conn
 	service, err := conn.CreateService(&gofastly.CreateServiceInput{
 		Name:    d.Get("name").(string),
-		Comment: "Managed by Terraform",
+		Comment: d.Get("name").(string),
```

Comment:
Shoudn't this be ` d.Get("comment").(string)`

Reply:
@matpimenta you're absolutely right! As you can see in the tests the current implementation is still working! I guess I was testing something and forgot to change it. Thanks for catching this one.

Reply:
Looks good now it is referring to comment :)